Sample Data/ASIM/Microsoft_Windows_AuditEvent_IngestedLogs.csv

```diff
@@ -0,0 +1,2682 @@
+"00000000-0000-0000-0000-000000000000","4/29/2023, 1:07:14.806 PM",OpsManager,"KUSTOWORKS\DC1$",Machine,"DC1.kustoworks.com","Microsoft-Windows-Security-Auditing",Security,1,0,"<EventData xmlns=""http://schemas.microsoft.com/win/2004/08/events/event"">
```

Comment:
Schema of this csv will not be able to accommodate both Security and Windows Event format. Please raise a separate file for both of them

Reply:
Sorted also for Event